ui/src/pages/version/components/forms/steps/ModelStep.js

```diff
@@ -12,12 +12,22 @@ import { DeploymentConfigPanel } from "../components/DeploymentConfigPanel";
 import { EnvVariablesPanel } from "../components/EnvVariablesPanel";
 import { LoggerPanel } from "../components/LoggerPanel";
 import { ResourcesPanel } from "../components/ResourcesPanel";
+import { ImageBuilderSection } from "../components/ImageBuilderSection";
 
 export const ModelStep = ({ version, isEnvironmentDisabled = false }) => {
   const { data, onChangeHandler } = useContext(FormContext);
   const { onChange } = useOnChangeHandler(onChangeHandler);
   const { errors } = useContext(FormValidationContext);
 
+  // isEnvironmentDisabled is true for redeployment and false for deploy 
+  const imagebuilderSection = !isEnvironmentDisabled ? (
```

Comment:
Hmmm.. If the first/previous deployment attempt failed, we will still go through the "redeployment" flow. In this case, `isEnvironmentDisabled` will be true but we may still want to show the image builder section to let the user tweak those values, in case that's where the prior deployment attempt failed.

So I'm thinking we can just show this section always.

Reply:
` we will still go through the "redeployment" flow` oh right, i was actually under the impression user have to create multiple new version to tweak this config. Just occur to me that they can call deploy multiple times, as long as the image isn't built, it will go through this flow